LayoutTests/compositing/overflow/scrolling-without-painting.html

```diff
@@ -20,17 +20,22 @@
     }
   </style>
   <script type="text/javascript">
-      window.addEventListener('load', function() {
+      window.addEventListener('load', async function() {
           if (!window.testRunner || !window.internals) {
               alert('This test requires testRunner to run!');
               return;
           }
           testRunner.dumpAsText(false);
+          testRunner.waitUntilDone();
+
+          await new Promise(resolve => requestAnimationFrame(resolve));
```

Comment:
You have to be careful with this. The promise gets resolved in the microtask at the end of requestAnimationFrame callbacks, which is in the middle of `Page::updateRendering()`. Maybe you want `await UIHelper.renderingUpdate()` instead, which bounces to the next runloop?

Reply:
That's probably a good idea, switched to that everywhere.